core/src/main/java/de/danoeh/antennapod/core/syndication/util/SyndTypeUtils.java

```diff
@@ -38,4 +48,19 @@ public static String getValidMimeTypeFromUrl(String url) {
 		}
 		return null;
 	}
+
+	/**
+	 * Should be used if mime-type of enclosure tag is not supported. This
+	 * method will return the mime-type of the file extension.
+	 */
+	public static String getMimeTypeFromUrl(String url) {
+		if (url != null) {
```

Comment:
I would change this to ``if(url == null) { return null; }``
less indentation, code looks a lot cleaner